libethash-cl/CLMiner.h

```diff
@@ -74,6 +74,7 @@ class CLMiner: public Miner
 		}
 	}
 	HwMonitor hwmon() override;
+	static const char *strClError(const cl_int err);
```

Comment:
There is not point of having this function public. Make it an static-linkage function in CLMiner.cpp. Also, do not declare argument with `const` in this case.

Reply:
Ok, I didn't like it very much inside the class myself. Why do you want me to drop the ``const``?

Reply:
Because you pass the argument by value. `const` is confusing as it suggests passing by reference.

Reply:
Afaik if you pass const as an argument, the compiler has more options to optimize the section of the code. That's why I put a const there in the first place.

Reply:
This is not true. The compiler does not care about your `const`.

Reply:
@chfast Thanks for merging, but I am still not sure why you don't like the ``const`` as the argument. Even if the compiler does not optimize it away (which imho he might, even if it's unlikely), it protects from stupidity like this happening later in the project: http://cpp.sh/5rwau
I really don't mean to criticize you here, but one more thing I learned myself and I can only so much confirm: https://stackoverflow.com/questions/117293/use-of-const-for-function-parameters/117557#117557